src/docfx/lib/json/YamlUtility.cs

```diff
@@ -22,7 +22,12 @@ internal static class YamlUtility
 
         public static string ReadMime(TextReader reader)
         {
-            return ReadMime(reader.ReadLine());
+            var mime = ReadMime(reader.ReadLine());
+            if (string.Compare(mime, "YamlDocument", StringComparison.OrdinalIgnoreCase) == 0)
+            {
+                return ReadDocumentType(reader.ReadLine());
```

Comment:
Im not sure if documenttype ia always the first property

Reply:
right